decentraland/unity-explorer
# Missing street from (-51,-51) to (-51,-1) (#2578)

## What does this PR change?

RoadData updated to change the rotation of the -51,-49 prefab from 270 to 90. Road prefab -70,-1 adapted so it connects with the other road forming a T. Two internal prefabs replaced with road prefabs of 1 corner.

## How to test the changes?

1. Launch the explorer
2. Go to -51,-49.
3. Check that the road that connects that point with -51,-1 is there and is properly connected with the other road.

![imagen](https://github.com/user-attachments/assets/80996bb9-4457-46a0-9f33-98642e80395c)
